editor/jest.config.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
         '\\.[jt]sx?$': 'babel-jest',
       },
       roots: ['src'],
-      transformIgnorePatterns: ['<rootDir>/node_modules/.pnpm/(?!file+..+)'],
+      transformIgnorePatterns: ['<rootDir>/node_modules/.pnpm/(?!file+..+|yjs|lib0)'],
```

Comment:
What's the deal with this? 👀

Reply:
That's so that the `yjs` and `lib0` libraries do get transformed to make them work with jest's module loader, as they're ES modules from memory which doesn't otherwise work.

Reply:
good point, @seanparsons could you leave a comment there saying so? and the same for globalThis.crypto in the other file

Reply:
Added some explanatory comments in e62ed6363.